python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -262,6 +263,18 @@ def run(self, splash, version, **kwargs):
         f.close()
         app.setStyleSheet(css)
 
+        # Initialize all of this after the style-sheet has been applied to any prompt are also
+        # styled after the Shotgun Desktop's visual-style.
+        splash.set_message("Initializing browser integration.")
+        try:
+            desktop_server_framework.launch_desktop_server()
+        except Exception:
+            logger.exception("Unexpected error while trying to launch the browser integration:")
+
+        # hide the splash if it exists
```

Comment:
Moved up, sometimes the splash screen would still be visible when the desktop window appeared.